kbase-extension/static/kbase/js/common/cellComponents/fieldTableCellWidget.js

```diff
@@ -35,7 +35,11 @@ define([
                 description: 'Field bus',
             }),
             inputControlFactory = config.inputControlFactory,
-            spec = config.parameterSpec;
+            spec = config.parameterSpec,
+            state = {
```

Comment:
Add a message state so that when `clearMessage` is called, it knows whether or not to clear the error class around the whole widget. (i.e. if `clearMessage` is called from the validation result, but there's still a duplicate error, the error class shouldn't get cleared).